frontend/src/features/admin-form/common/components/AdminFormNavbar/AdminFormNavbar.tsx

```diff
@@ -142,6 +143,17 @@ export const AdminFormNavbar = ({
         />
         <Box display={{ base: 'none', md: 'flex' }}>
           <ButtonGroup spacing="0.5rem" isDisabled={!formInfo}>
+            <Tooltip label="Help">
+              <IconButton
+                aria-label="Help"
+                variant="outline"
+                onClick={(e) => {
+                  e.preventDefault()
+                  window.open('https://guide.form.gov.sg/')
```

Comment:
We had a [recent PR](https://github.com/opengovsg/FormSG/pull/4499) where all links were transferred to a constant file. Let's do the same for this one, please.

Reply:
done, apologies for not being aware of the latest changes! 🙏

Reply:
No worries! Thanks for the contribution as always 🙏 🎉